src/DynamoCoreWpf/Properties/Resources.resx

```diff
@@ -2311,4 +2311,13 @@ Uninstall the following packages: {0}?</value>
   <data name="PreferencesViewVisualSettingsTab" xml:space="preserve">
     <value>Visual Settings</value>
   </data>
+  <data name="PreferencesWindowFontSizes" xml:space="preserve">
+    <value>Small,Medium,Large,Extra Large</value>
+  </data>
+  <data name="PreferencesWindowLanguages" xml:space="preserve">
+    <value>English,Czech,German,Spanish,French,Italian,Japanese,Korean,Polish,Portuguese,Brazilian,Russian,Chinese Simplified,Chinese Traditional</value>
+  </data>
+  <data name="PreferencesWindowNumberFormats" xml:space="preserve">
+    <value>0,0.0,0.00,0.000,0.0000</value>
```

Comment:
Same problem here, these become duplicates to `DynamoViewSettingMenuNumber00000` `DynamoViewSettingMenuNumber0000` etc

Reply:
@QilongTang  I removed the duplicated strings in the next commit:
https://github.com/DynamoDS/Dynamo/pull/11542/commits/89b81d746df5a9d76bb3597c2f49446c392abae8